src/protocols/rdp/channels/rdpdr/rdpdr-fs-messages-file-info.c

```diff
@@ -135,6 +135,10 @@ void guac_rdpdr_fs_process_set_rename_info(guac_rdp_common_svc* svc,
     wStream* output_stream;
     char destination_path[GUAC_RDP_FS_MAX_PATH];
 
+    /* Check stream size prior to reading. */
+    if (Stream_GetRemainingLength(input_stream) < 6)
+        return;
```

Comment:
Is it worth logging a warning when a received PDU is unexpectedly small? Should make it easier to trace down the cause of unexpected behavior due to RDP protocol violations.

Reply:
Yeah, I can add these in.

Reply:
Okay, warning logging has been added in - let me know if I missed anything or if the warning messages need more detail, etc.